pyHomee/__init__.py

```diff
@@ -541,11 +541,11 @@ def ws_url(self) -> str:
 
         return f"ws://{self.host}:7681"
 
-    async def wait_until_connected(self) -> Coroutine[Any, Any, Literal[True]]:
+    async def wait_until_connected(self) -> Literal[True]:
         """Return a coroutine that runs until a connection has been established."""
         return await self._connected_event.wait()
 
-    async def wait_until_disconnected(self) -> Coroutine[Any, Any, Literal[True]]:
+    async def wait_until_disconnected(self) -> Literal[True]:
         """Return a coroutine that runs until the connection has been closed."""
         return await self._disconnected_event.wait()
```

Comment:
`self._disconnected_event.wait()` is awaited before it's returned. Thus the return type should only be `Literal[True]`.